generator/testdata/rust/protobuf/golden/iam/v1/src/lib.rs

```diff
@@ -55,8 +54,3 @@ pub(crate) mod info {
     }
 }
 
-/// A `Result` alias where the `Err` case is an [Error].
-pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
I do understand not wanting a chain of aliases, but maybe those aliases in gax should just not exist. I think the downside of this change is there is no longer a top-level links to these important types in our crates documentation -- outside of method signatures. Also, for result specifically it does feel a little odd to return another crates result type.